lib/AST/Module.cpp

```diff
@@ -493,9 +493,6 @@ ArrayRef<ImplicitImport> ModuleDecl::getImplicitImports() const {
                            {});
 }
 
-bool ModuleDecl::isClangModule() const {
-  return findUnderlyingClangModule() != nullptr;
-}
```

Comment:
The corresponding method declaration in `include/swift/AST/Module.h` should also be deleted.